packer-plugin-sdk/template/doc.go

```diff
@@ -0,0 +1,49 @@
+/*
+Package template helps plugins parse the Packer template into golang structures.
+
+This package should be imported and used by all plugins. It implements the
+golang template engines that Packer documentes on its website, along with input
+validation, custom type decoding, and template variable interpolation.
+
+A simple usage example that defines a config and then unpacks a user-provided
+json template into the provided config:
+
+```golang
+import (
```

Comment:
If you add 2 spaces at the left of consecutive lines the lines will be shown in a code block:

https://godoc.org/github.com/fluhus/godoc-tricks

Reply:
ooo. I'll do that.